lms/djangoapps/verify_student/management/commands/tests/test_manual_verify_student.py

```diff
@@ -77,7 +77,7 @@ def test_manual_verifications_created_date(self):
             created_at__gte=earliest_allowed_verification_date()
         )
 
-        self.assertQuerysetEqual(verification1, [repr(r) for r in verification2])
+        self.assertQuerysetEqual(verification1, [repr(r) for r in verification2], transform=repr)
```

Comment:
TransactionTestCase.assertQuerysetEqual() no longer calls repr() on a queryset when compared to string values.

Reply:
As a solution, it is necessary to explicitly assign transform attribute now.